test/commands/codepush/release-cordova-test.ts

```diff
@@ -0,0 +1,415 @@
+import * as Sinon from "sinon";
+import * as Nock from "nock";
+import CodePushReleaseCordovaCommand from "../../../src/commands/codepush/release-cordova";
+import { CommandArgs } from "../../../src/util/commandline/command";
+import { CommandFailedResult, CommandResult } from "../../../src/util/commandline/command-result";
+import { expect } from "chai";
+import * as which from "which";
+import * as cp from "child_process";
+import * as fs from "fs";
+import * as validationUtils from "../../../src/commands/codepush/lib/validation-utils";
+import * as path from "path";
+
+describe("Codepush release-cordova command", function () {
```

Comment:
```suggestion
describe("codepush release-cordova command", function () {
```
Just to make it consistent with other tests. 

PS `release-react-test.ts` should be fixed as well.

Reply:
and all tests in deployment folder should be changed, or we can chose `CodePush` option